src/app.js

```diff
@@ -117,6 +119,23 @@ function getOptions() {
   return options;
 }
 
+/**
+ * The command parser talks to the server over the public HTTP/WS API,
+ * the gateway needs to configure it with a JWT and a server address
+ * @param {http.Server|https.Server} server
+ */
+function commandParserConfigure(server) {
```

Comment:
I'm not sure whether this logic should live in app.js or in `CommandsController` itself, by just passing the server object to `CommandsController` directly, or via `Router.configure()`. Maybe it is better to inject the jwt rather than have `CommandsController` import `JSONWebToken` itself though, what are you doing for the rules engine @hobinjk ?

Reply:
Reviewing #263 and I realised this code was copied from there :) Will discuss in #263